server/tests/common/data.py

```diff
@@ -211,3 +215,167 @@ async def _alice_generated_realm_data(now: DateTime) -> None:
             assert False, unknown
 
     return _alice_generated_realm_data
+
+
+Http46xTesterDoCallback = Callable[[], Coroutine[None, None, None]]
+Http46xTester = Callable[[Http46xTesterDoCallback], Coroutine[None, None, None]]
+
+
+# TODO: not sure how to test "organization_not_found"
+@pytest.fixture(params=("organization_expired", "author_revoked", "author_frozen"))
+async def http_46x_authenticated_tester(
```

Comment:
I'd rename this functions (See comment below in `test_api.py`). Also I would put family first, I think is clearer when using it as fixture.

Something like:
```suggestion
async def authenticated_common_errors_tester(
```